torch/_inductor/fx_passes/quantization.py

```diff
@@ -521,6 +532,9 @@ def __init__(self, op_name: str, scalars_attr=None, algorithm_attr=None):
             UnaryAttr("relu", [], ""): generate_pattern_with_unary(
                 dequantize_qconv_pt2e_pattern, aten.relu.default
             ),
+            UnaryAttr("hardtanh", [], ""): _hardtanh_fusion(
+                dequantize_qconv_pt2e_pattern
+            ),
```

Comment:
Can we extend `generate_pattern_with_unary` to support "hardtanh" fusion rather than relying on a pattern specific function?